doc/bibliography.html

```diff
@@ -453,6 +453,11 @@ <H2>Bibliography</H2>
 <em>Data Structures for Weighted Matching and Nearest Common Ancestors with Linking</em><br>
 Proceedings of the First Annual ACM-SIAM Symposium on Discrete Algorithms, pp. 434-443, 1990.
 
+<p></p><dt><a name="tarjan83">77</a>
+<dd>Robert Endre Tarjan<br>
+<em>Linking and Cutting Trees</em><br>
+Data Structures and Network Algorithms, ISBN: 978-0-89871-187-5, pp. 59-70, 1983.
+
```

Comment:
Wow, I'm surprised that this is the first time that this book is referenced -- probably long overdue!

Reply:
Thanks for helping review :smile: 
And enjoy parenting! :baby_bottle: